cli/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ _Released 4/16/2024 (PENDING)_
 **Misc:**
 
 - Updated the Chrome flags to not show the "Enhanced Ad Privacy" dialog. Addresses [#29199](https://github.com/cypress-io/cypress/issues/29199).
+- Suppresses benign warnings that reference Vulkan on GPU-less Docker hosts. Addresses [#29085](https://github.com/cypress-io/cypress/issues/29085). Addressed in [#29278](https://github.com/cypress-io/cypress/pull/29278).
```

Comment:
The issue affects environment without Docker, so you could probably just say

"on GPU-less hosts." 

and leave out the word "Docker".

Reply:
Thank you! Addressed in https://github.com/cypress-io/cypress/pull/29278/commits/48b406d9ad6bfe4c61b3b6cee3aa4cad9f66c80d